packages/catalog-model/src/kinds/GroupEntityV1alpha1.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as yup from 'yup';
+import type { Entity } from '../entity/Entity';
+import type { EntityPolicy } from '../types';
+
+const API_VERSION = ['backstage.io/v1alpha1', 'backstage.io/v1beta1'] as const;
+const KIND = 'Group' as const;
+
+export interface GroupEntityV1alpha1 extends Entity {
+  apiVersion: typeof API_VERSION[number];
+  kind: typeof KIND;
+  spec: {
+    type: string;
+    owner: string;
+    parent?: string[];
+    ancestors?: string[];
+    children?: string[];
+    grandchildren?: string[];
```

Comment:
I know the suggestion said `grandchildren`, but I argued with the others here and arrived at `descendants` is the correct term :) Would you mind changing it?

Reply:
Oh one more thing. Make `ancestors`, `children` and `descendants` all non-optional! There's no value in being able to discern between `undefined` and `[]`.